pygmt/clib/session.py

```diff
@@ -494,6 +494,97 @@ def get_default(self, name):
 
         return value.value.decode()
 
+    def get_common(self, option):
```

Comment:
Please add this function to `doc/api/index.rst`.

Reply:
Done in 9fb950db28755243a3785bd67d464f747579bc34.